package.json

```diff
@@ -18,15 +18,21 @@
     "e2e:cloud": "E2E_RUN_IN_CLOUD=true npm run e2e",
     "e2e:tunnel": "E2E_BASE_URL=http://localhost:3000 E2E_USE_TUNNEL=true npm run e2e:cloud",
     "e2e:local": "E2E_BASE_URL=http://localhost:3000 npm run e2e",
+    "jest": "./node_modules/jest/bin/jest.js",
+    "jest:update": "./node_modules/jest/bin/jest.js -u",
```

Comment:
Nice call on adding this option. I always forget what the specific command is to update. That'll save some time. 💯